CppCoreGuidelines.md

```diff
@@ -385,7 +385,7 @@ A much clearer expression of intent would be:
         string val;
         cin >> val;
         // ...
-        auto p = find(v, val);  // better
+        auto p = find(begin(v), end(v), val);  // better
```

Comment:
Ok, thought that might've been the intention. Do you think the mention of `std::find` just above should reflect that? Just dropping the `std::` for example.